orttraining/orttraining/models/runner/training_util.h

```diff
@@ -54,6 +56,26 @@ class DataSet {
 
   void RandomShuffle();
 
+  /**
+     The method is for getting model training params that are part of training data
+     first load .onnx model in Netron to get the mapping between input data and the graph
+     for example, a bert model (see input_name_map in bert/main.cc) requires 7 inputs
+     each input may have different tensor shape, like so
+        intput1 : int64[batch,sequence]
+        masked_lm_ids:  int64[batch,dynamic_prediction_count]
+     When loading training data, the actual shape vector of tensor would not include "batch", thus caller needs to adjust 
+     the index position (i.e., subtract by 1) to get the correspondent value. For example,
+     to get sequence length, we can look for input name "input1" and get its value in shape vector's position 0 (NOT 1) element 
+     based on input_to_dimension_mapping (see input_to_dimension_mapping example in bert/main.cc) to map the name with the vector position, 
+     like so
+        {"input1", {"SeqLen", 0}}  => sequence->SeqLen , where SeqLen will be populated as key in mapped_dimensions 
+     @param input_to_dimension_mapping tensor shape dimension mapping from training data, example above {"input1", {"SeqLen", 0}} to map 
+                        input1's "sequence" at position 0 into "SeqLen" as mapped_dimensions key
+     @param mapped_dimensions populated as json for perf monitoring
```

Comment:
the description for mapped_dimensions describes how this is used more than what it actually contains